std/experimental/allocator/common.d

```diff
@@ -92,13 +92,14 @@ size_t goodAllocSize(A)(auto ref A a, size_t n)
 Returns s rounded up to a multiple of base.
 */
 @safe @nogc nothrow pure
-package size_t roundUpToMultipleOf(size_t s, uint base)
+size_t roundUpToMultipleOf(size_t s, uint base)
```

Comment:
I made it package-visible because it's in the wrong place. Once we expose it we'll need to do it forever. It should be shuttled to the right place. I'd say put it in `std.numeric`. Other possible candidates: `std.conv`, `std.math`.